designs/ble-mote.stanza

```diff
@@ -118,77 +112,143 @@ pcb-module sensor-mote :
   val hum-tps = add-testpoint([   hum.i2c.sda
                                   hum.i2c.scl], Testpoint-SMDLoop)
 
-  val pms-tps = add-testpoint([   PC-TX
-                                  PC-RX
+  val pms-tps = add-testpoint([   particle-counter-interface.uart.tx
+                                  particle-counter-interface.uart.tx
                                   gnd ], Testpoint-SMDLoop)
- 
 
-  ; Add a ground plane
-  geom(gnd):
-    copper-pour(LayerIndex(1), isolate = 0.1, rank = 1) = board-shape
-    copper-pour(LayerIndex(2), isolate = 0.1, rank = 1) = board-shape
-
-; Add artwork
-  inst pcb-name: ocdb/board-text/text("PCB, BLE MOTE", 1.5, clearance(current-rules(), MinSilkscreenWidth) )
-  inst ver-rev : ocdb/board-text/text("v0 rev0", 1.0, 0.0)
-  inst logo : ocdb/jitx-logo/small(Silkscreen("artwork", Top),true)
-  inst assy-label : ocdb/mechanical/gen-label()
-  place(pcb-name) at loc(0.0,-0.5) on Bottom
-  place(ver-rev) at loc(10.0,3.0) on Bottom
-  place(logo) at loc(6.0,0.0, 0.0) on Bottom
-  place(assy-label) at loc(5.0,-5.25, 0.0) on Bottom
-
-  ; Apply groups to sub-circuits for schematic and layout
-  schematic-group([proc, ant, debug-tps rgb, i2c-pullups]) = esp32
   schematic-group([usb, ldo, cp2105, power-tps containing-instance(protected-usb) as JITXObject]) = power
   schematic-group([hum hum-tps]) = humidity-sensor
   schematic-group([particle-counter pms-tps]) = particle-counter
 
-  layout-group([proc, ant]) = proc
   layout-group([usb, ldo, cp2105 containing-instance(protected-usb) as JITXObject]) = power
   layout-group([hum hum-tps]) = hum
   layout-group(particle-counter) = particle-counter
 
-  net VBUS (usb.usb-2.vbus.vdd)
-  net D- (usb.usb-2.data.N)
-  net D+ (usb.usb-2.data.P)
-
-  property(VBUS.net-voltage) = min-max(4.8, 5.2)
-  property(P5V0.net-voltage) = min-max(4.9, 5.1)
-  property(P3V3.net-voltage) = typ(3.3)
-  property(gnd.net-voltage) = typ(0.0)
-
   ; Set up netclasses
   val power-net-class = NetClass(`Power, [`min-trace => 0.50])
```

Comment:
Learn something new everyday -- a property called `power-net-class` with a `NetClass()` assignor. But you use symbols (``) in the `NetClass()` function?

Reply:
Yes - I'll make a note to add docs for this.